cmd/commons/application_config.go

```diff
@@ -11,16 +11,19 @@ import (
 	"sync"
 )
 
+// ServerCoordinates describes server configuration parameters
 type ServerCoordinates struct {
-	ApiURL   string `json:"apiUrl"`
+	APIURL   string `json:"apiUrl"`
```

Comment:
I like the `ApiUrl` a lot more as it is much easier to read. But GoLang lining rules think otherwise :( 
Imagine field called `APIJSONURLID` :P instead of `apiJsonUrlId`